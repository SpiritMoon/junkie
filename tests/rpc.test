#!/bin/sh

NAME="RPC"
PCAPS="pcap/nfs/nfs.pcap"
filter() {
	grep '^RPC'
}
OUTPUT="rpc.out"

. "$srcdir/dump_test"
