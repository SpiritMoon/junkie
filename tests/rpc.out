-- pcap/nfs/nfs.pcap --
RPC: type: CALL, rpc version: 2, program: 100003, program version: 3, procedure: 0
RPC: type: REPLY, reply status : MSG_ACCEPTED
RPC: type: CALL, rpc version: 2, program: 100003, program version: 3, procedure: 1
RPC: type: REPLY, reply status : MSG_ACCEPTED
